operators/src/adapters/raster_subquery/raster_subquery_reprojection.rs

```diff
@@ -237,8 +260,7 @@ pub fn fold_by_coordinate_lookup_impl<T>(
 where
     T: Pixel,
 {
-    const OVERPROVISIONING: usize = 1; // split the work into multiple chunks per thread.
-    const MAX_PARALELLISM: usize = 16; // max task / work split
+    const MIN_ROWS_IN_PAR_CHUNK: usize = 32; // this must never be smaller than 1
```

Comment:
Here, the size differs from above. But why? Is this empirically determined?

Reply:
yes thats the 8/16 split from the graph i generated.